packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.types.ts

```diff
@@ -22,6 +22,12 @@ export interface IFocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement
    */
   componentRef?: IRefObject<IFocusTrapZone>;
 
+  /**
+   * If set, the FocusTrapZone's focus trapping behavior will be disabled.
```

Comment:
Suggestion based on the [wiki](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Commenting-conventions):

```suggestion
   * Disables the FocusTrapZone's focus trapping behavior if set.
```